src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -6154,6 +6156,8 @@ def update_vnet(cmd, instance, vnet_prefixes=None, dns_servers=None, ddos_protec
         instance.ddos_protection_plan = None
     elif ddos_protection_plan is not None:
         instance.ddos_protection_plan = SubResource(id=ddos_protection_plan)
+    if flowtimeout:
```

Comment:
Can flowtimeout be 0? If so, you should change the condition expression

```suggestion
    if flowtimeout is not None:
```